circuit/program/src/data/plaintext/to_bits.rs

```diff
@@ -23,10 +23,12 @@ impl<A: Aleo> ToBits for Plaintext<A> {
             Self::Literal(literal, bits_le) => {
                 // Compute the bits of the literal.
                 let bits = bits_le.get_or_init(|| {
-                    let mut bits_le = vec![Boolean::constant(false), Boolean::constant(false)]; // Variant bit.
+                    let mut bits_le = Vec::new();
```

Comment:
note: this is a conservative tweak which ensures that the vector starts out with a capacity larger than 2 elements